src/xdocs/checks.xml

```diff
@@ -48,6 +48,10 @@
         <td>Check location of annotation on language elements.</td>
       </tr>
       <tr>
+        <td><a href="config_annotation.html#AnnotationOnSameLine">AnnotationOnSameLine</a></td>
+        <td>Check annotation on language elements is on same line with its target element.</td>
```

Comment:
Why isn't this the same text as in the check in `config_annotation` or in the main Javadoc for the check?

Reply:
Updated to javadoc text